integration/airflow/tests/integration/requests/athena.json

```diff
@@ -283,6 +283,105 @@
     "run": {
         "facets": {}
     }
+},{
+    "eventType": "START",
+    "inputs": [],
+    "job": {
+        "facets": {
+            "documentation": {
+                "description": "Determines the popular day of week orders are placed."
+            },
+            "sql": {
+                "query": "\n    SELECT * FROM openlineage_integration_{{ env_var('AWS_ATHENA_SUFFIX') }}.test_orders;\n\n    "
+            }
+        },
+        "name": "athena_dag.athena_select_from_other_db",
+        "namespace": "food_delivery"
+    },
+    "outputs": [
+        {
+            "facets": {
+                "dataSource": {
+                    "name": "awsathena://athena.{{ env_var('AWS_DEFAULT_REGION') }}.amazonaws.com",
```

Comment:
We actually don't have Athena instance setup to run integration tests for it but I strongly believe this should be an input rather than output. It's same for COMPLETE event as well.

Reply:
Yes you are right. I am trying to get those test running locally on my PC but I faced a lot of problems. I will commit the good version once I can run test locally

Reply:
[This](https://openlineage.io/docs/development/developing/python/setup) and [this](https://openlineage.io/docs/development/developing/python/tests/airflow) doc pages might help you.

Reply:
Thanks! It seems like there is an error in the [Dockerfile](https://github.com/OpenLineage/OpenLineage/blob/main/integration/airflow/tests/integration/Dockerfile#L6) at line 6
I replaced locally
```python
COPY integration/target /app/openlineage/integration/sql/target
```
with
```python
COPY integration/sql/target /app/openlineage/integration/sql/target
```

I can commit the change

Reply:
@rinzool there have been recent changes to CI introduced. Above should be updated for dev environment as well but let me figure this out properly some time later.